moveit_core/trajectory_processing/src/ruckig_traj_smoothing.cpp

```diff
@@ -45,59 +45,59 @@ namespace trajectory_processing
 {
 namespace
 {
-const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_trajectory_processing.ruckig_traj_smoothing");
-constexpr double DEFAULT_MAX_VELOCITY = 5;           // rad/s
-constexpr double DEFAULT_MAX_ACCELERATION = 10;      // rad/s^2
-constexpr double DEFAULT_MAX_JERK = 20;              // rad/s^3
-constexpr double IDENTICAL_POSITION_EPSILON = 1e-3;  // rad
-constexpr double MAX_DURATION_EXTENSION_FACTOR = 5.0;
-constexpr double DURATION_EXTENSION_FRACTION = 1.1;
-constexpr double MINIMUM_VELOCITY_SEARCH_MAGNITUDE = 0.01;  // rad/s. Stop searching when velocity drops below this
+rclcpp::Logger const LOGGER = rclcpp::get_logger("moveit_trajectory_processing.ruckig_traj_smoothing");
```

Comment:
Also, should we really have magic default values in here? I'd argue that it would probably be safer to just require configured limits

Reply:
I think the default (vel/accel/jerk) values do make sense. There are a ton of robots out there that don't even have manufacturer-specified acceleration limits and I don't think any of them have jerk limits specified. It's nice to have these safe default values that just work nicely and prevent hardware damage without the user needing to do anything.

If we wanted to move these parameters to a yaml file, where would they even go? I guess they would belong in `ompl_planning.yaml` because that's where the Ruckig plugin is specified.

Reply:
If there are standard magic values, it could be nice to info / warn users about those being used.

There have been many questions on ROS Answers along the lines of "why is my robot so slow when using MoveIt?", which almost always turned out to be due to the time-parameterisation algorithms using the default `1.0m/s` and/or `1.0m/s^2` limits hard-coded in them (without any mention of this to the user).

Reply:
OK, I'll add a helpful hint if no vel/accel/jerk limit is defined.